eth/transaction_signing_test.go

```diff
@@ -0,0 +1,92 @@
+package eth_test
+
+import (
+	//    "strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/INFURA/go-ethlibs/eth"
+)
+
+func TestSignTransaction(t *testing.T) {
+	data := []byte("0x")
```

Comment:
See my previous comments on `NewTransaction`, but this seems really odd to be passing a string as a byte array.